common/src/registers/MachRegister.C

```diff
@@ -100,16 +100,8 @@ namespace Dyninst {
     }
   }
 
-  std::string MachRegister::name() const {
-    assert(names() != NULL);
-    NameMap::const_iterator iter = names()->find(reg);
-
-    std::string ret;
-    if(iter != names()->end()) {
-      return iter->second;
-    }
-    common_parsing_printf(" can't find register with index %x\n", (unsigned int)reg);
-    return std::string("<INVALID_REG>");
+  std::string const& MachRegister::name() const {
+    return names.find(reg)->second;
   }
```

Comment:
@hainest, doesn't this need to check for reg not being in the map?  Couldn't someone use `MachRegister::MachRegister(bogusRegId)`, and bogusRegId would not be in `names`?